components/Article.vue

```diff
@@ -51,19 +51,21 @@
         return 'cf-' + this.content.sys.id;
       },
 
-      // Allows us to smoothly transition between unknown article heights and
-      // a known minimum. Returns 'auto' when no transition is possible.
+      // Allows us to smoothly transition between unknown min-max heights.
+      // Returns 'auto' when no transition is necessary.
       getArticleHeight() {
         if (!this.isExpandable) {
           return 'auto'
         } else {
-          return this.isExpanded ? this.articleHeight : '280px';
+          return this.isExpanded ? this.articleHeight + 'px' : this.articleMinHeight + 'px';
```

Comment:
Switched all values to be `Number`s and I'm only appending `px` once we print the inlline CSS.